docroot/modules/custom/mass_media/mass_media.module

```diff
@@ -28,6 +29,10 @@ use Drupal\media\MediaInterface;
  * @throws \Drupal\Core\Entity\EntityStorageException
  */
 function mass_media_media_update(EntityInterface $entity) {
+  if (!($entity instanceof DocumentBundle)) {
+    return;
+  }
+
```

Comment:
What is the purpose of this? Maybe add a code comment. It doesnt look like the subsequent code is specific to document bundle.

Reply:
It is specific to document bundle only.
// getUploadFile method and field_upload_file field are only available to documents.
So the condition added to prevent logs thrown for other bundles.

Reply:
That implies that video media remain available at public urls even after they are unpublished. Is this a problem, @joeg8612?

Reply:
Regardless of that answer, the issue is fixed so merging.